go_covert_lib/client/src/App.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState, useEffect } from 'react';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import FormControl from 'react-bootstrap/FormControl';
+import InputGroup from 'react-bootstrap/InputGroup';
+import FormCheck from 'react-bootstrap/FormCheck';
+import './styles.css';
+
+/**
+ * IMPORTANT NOTE: For styling, refer to https://getbootstrap.com/docs/4.0/utilities/position/
+ */
+
+const App = () => {
+    const [textToSend, setTextToSend] = useState("");
```

Comment:
You will see a lot of `useState` and `useEffect` in the frontend throughout development. These are called hooks and are a core part of new React design (16.8+). A great & short intro to them is available here, I highly recommend taking at least a light read through:
https://reactjs.org/docs/hooks-intro.html